docs_user/modules/proc_deploying-a-ceph-ingress-daemon.adoc

```diff
@@ -48,14 +52,14 @@ haproxy-bundle successfully restarted
     * haproxy-bundle-podman-2   (ocf:heartbeat:podman):  Started controller-2
 ----
 
-. Confirm that no process is bound to 8080:
+. Confirm that no process is connected to the 8080 port:
```

Comment:
"Confirm that no process is connected to port 8080:"

Maybe "bound" is ok here - like it is to do with port binding - could check with SME